web/src/SetBoundaryMode.svelte

```diff
@@ -43,8 +43,9 @@
           mode: "neighbourhood",
         };
       } catch (err) {
+        console.log("error when setting hand drawn neighborhood", err);
         window.alert(
-          "Known georust bug hit, sorry. You may need to just refresh the page now.",
+          "Sorry, an error occurred. You may need to refresh the page for the application to continue working.",
```

Comment:
Stop tarnishing the brand! 🤣 

(truly - I think the old message is no longer relevant, so I made it more generic)